rpc/core/routes.go

```diff
@@ -14,24 +14,25 @@ var Routes = map[string]*rpc.RPCFunc{
 	"unsubscribe_all": rpc.NewWSRPCFunc(UnsubscribeAll, ""),
 
 	// info API
-	"health":               rpc.NewRPCFunc(Health, ""),
-	"status":               rpc.NewRPCFunc(Status, ""),
-	"net_info":             rpc.NewRPCFunc(NetInfo, ""),
-	"blockchain":           rpc.NewRPCFunc(BlockchainInfo, "minHeight,maxHeight"),
-	"genesis":              rpc.NewRPCFunc(Genesis, ""),
-	"block":                rpc.NewRPCFunc(Block, "height"),
-	"block_by_hash":        rpc.NewRPCFunc(BlockByHash, "hash"),
-	"block_results":        rpc.NewRPCFunc(BlockResults, "height"),
-	"commit":               rpc.NewRPCFunc(Commit, "height"),
-	"check_tx":             rpc.NewRPCFunc(CheckTx, "tx"),
-	"tx":                   rpc.NewRPCFunc(Tx, "hash,prove"),
-	"tx_search":            rpc.NewRPCFunc(TxSearch, "query,prove,page,per_page,order_by"),
-	"validators":           rpc.NewRPCFunc(Validators, "height,page,per_page"),
-	"dump_consensus_state": rpc.NewRPCFunc(DumpConsensusState, ""),
-	"consensus_state":      rpc.NewRPCFunc(ConsensusState, ""),
-	"consensus_params":     rpc.NewRPCFunc(ConsensusParams, "height"),
-	"unconfirmed_txs":      rpc.NewRPCFunc(UnconfirmedTxs, "limit"),
-	"num_unconfirmed_txs":  rpc.NewRPCFunc(NumUnconfirmedTxs, ""),
+	"health":                   rpc.NewRPCFunc(Health, ""),
+	"status":                   rpc.NewRPCFunc(Status, ""),
+	"net_info":                 rpc.NewRPCFunc(NetInfo, ""),
+	"blockchain":               rpc.NewRPCFunc(BlockchainInfo, "minHeight,maxHeight"),
+	"genesis":                  rpc.NewRPCFunc(Genesis, ""),
+	"block":                    rpc.NewRPCFunc(Block, "height"),
+	"block_by_hash":            rpc.NewRPCFunc(BlockByHash, "hash"),
+	"block_results":            rpc.NewRPCFunc(BlockResults, "height"),
+	"commit":                   rpc.NewRPCFunc(Commit, "height"),
+	"check_tx":                 rpc.NewRPCFunc(CheckTx, "tx"),
+	"data_availability_header": rpc.NewRPCFunc(DataAvailabilityHeader, "height"),
```

Comment:
So now to trigger DASing can we use dah endpoint instead of commit?

Reply:
If the client is run with the DAS parameters, _every_ request, that triggers an update will trigger DAS. 
I.e. search for `updateLightClientIfNeededTo` in this: https://github.com/lazyledger/lazyledger-core/blob/880ede182509cd2b7f248498e49f78a4c72414a9/light/rpc/client.go#L134